app-backend/backsplash-core/src/main/scala/com/rasterfoundry/backsplash/MosaicImplicits.scala

```diff
@@ -92,7 +94,7 @@ class MosaicImplicits[HistStore: HistogramStore](mtr: MetricsRegistrator,
           // the empty tile needs to be four band as well
           ndtile: MultibandTile = ArrayMultibandTile.empty(
             IntConstantNoDataCellType,
-            if (bandCount == 1) 4 else bandCount,
+            if (bandCount == 1) 1 else bandCount,
```

Comment:
Maybe just `bandCount`

Reply:
Ha yeah. Still a WIP lol -- the only relevant commit for this PR is the last one too btw